### PR TITLE
Capture log messages at the current log level and truncate if necessary DAT-6366

### PR DESCRIPTION
**Pull request by wwillard7800**
## Environment
**Liquibase Version**:

**Liquibase Integration & Version**: <Pick one: CLI, maven, gradle, spring boot, servlet, etc.>

**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**:

**Operating System Type & Version**:

## Pull Request Type

* [ ] Bug fix (non-breaking change which fixes an issue.)
* [ ] Enhancement/New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
A clear and concise description of the issue being addressed.  Additional guidance [here](https://liquibase.jira.com/wiki/spaces/LB/pages/1274904896/How+to+Contribute+Code+to+Liquibase+Core).

* Describe the actual problematic behavior.
* Ensure private information is redacted.

## Steps To Reproduce
List the steps to reproduce the behavior.

* Please be precise and ensure private information is redacted
* Include things like
  * Files used - sql scripts, changelog file(s), property file(s), config files, POM Files
  * Exact commands used - CLI, maven, gradle, spring boot, servlet, etc.

## Actual Behavior
A clear and concise description of what happens in the software **before** this pull request.

* Include console output if relevant
* Include log files if available.

## Expected/Desired Behavior
A clear and concise description of what happens in the software **after** this pull request.

## Fast Track PR Acceptance Checklist:

* [ ] Build is successful and all new and existing tests pass
* [ ] Added [Unit Test(s)](https://liquibase.jira.com/wiki/spaces/LB/pages/1274937609/How+to+Write+Liquibase+Core+Unit+Tests)
* [ ] Added [Integration Test(s)](https://liquibase.jira.com/wiki/spaces/LB/pages/1276608569/How+to+Write+Liquibase+Core+Integration+Tests)
* [ ] Added [Test Harness Test(s)](https://github.com/liquibase/liquibase-test-harness/pulls)
* [ ] Documentation Updated

## Need Help?
Come chat with us on our [discord channel](https://discord.com/channels/700506481111597066/700506481572839505)

┆Issue is synchronized with this [Jira Story](https://datical.atlassian.net/browse/LB-1166) by [Unito](https://www.unito.io/learn-more)

### CODE REVIEW

File: liquibase-core/src/main/java/liquibase/hub/HubUpdater.java [hunk 1]
```diff
@@ -169,7 +167,11 @@ public void postUpdateHub(Operation updateOperation, BufferedLogService bufferLo
 
             //
             // Send the COMPLETE operation event
+            // Capture the current log level to use for filtering
             //
+            java.util.logging.Logger liquibaseLogger = java.util.logging.Logger.getLogger("liquibase");
```
Comment (nvoxland) [inline liquibase-core/src/main/java/liquibase/hub/HubUpdater.java:172]: The logger people use is pluggable, so not necessarily the java logger like this.

**Review by nvoxland — CHANGES_REQUESTED** [review]
We can't check the level like you are doing with the java logger. 

The current logging APIs is designed for the logger to be configured outside our control, so there is no "get the current level" on purpose. Including for the reason that there really is no single "level". Console can have one level, a file can have a different level, and stuff in liquibase.snapshot can have a 3rd level.

So not sure this is a feature that is even possible?

File: liquibase-core/src/main/java/liquibase/hub/HubUpdater.java (hunk 1 above)
Comment (wwillard7800) [reply liquibase-core/src/main/java/liquibase/hub/HubUpdater.java:172]: I found an example in the code where we were still doing this, I thought it was plausible.

File: liquibase-core/src/main/java/liquibase/hub/HubUpdater.java [hunk 2]
```diff
@@ -223,6 +225,12 @@ public void postUpdateHubExceptionHandling(Operation updateOperation,
             if (updateOperation == null || hubIsNotAvailable(changeLog.getChangeLogId())) {
                 return;
             }
+
+            //
+            // Capture the current log level to use for filtering
+            //
+            java.util.logging.Logger liquibaseLogger = java.util.logging.Logger.getLogger("liquibase");
```
Comment (nvoxland) [inline liquibase-core/src/main/java/liquibase/hub/HubUpdater.java:232]: This needs to change still

**Review by nvoxland — CHANGES_REQUESTED** [review]

**Review by nvoxland — APPROVED** [review]

**Pull request merged**